nncf/torch/quantization/layers.py

```diff
@@ -347,6 +349,8 @@ def forward(self, x):
         # TODO: refactor to get rid of extra if's and calls on each forward
         if not self.is_enabled_quantization():
             return x
+        if self._old_level_range_setting:
+            self.set_level_ranges()
```

Comment:
It's should be run after load pretrained model.
But in this case levels will be incorrected while not run forward, it's affect to strip model if run `strip` after `load_state`.

Reply:
Nope, `strip` will not be affected `set_level_ranges` calls https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/quantization/strip.py#L75